docs/content/latest/yugabyte-platform/security/enable-encryption-in-transit.md

```diff
@@ -12,67 +12,170 @@ isTocNested: true
-2. Click **Add Certificate**. The **Add Certificate** dialog appears.
-3. Click **Upload Certificate File**, browse to the root certificate file (`<file-name>.crt`), and upload it.
-4. Click **Upload Key File**, browse to the root certificate file (`<file-name>.key`), and upload it.
-5. In **Name Certificate**, enter a meaningful name.
-6. In **Expiration Date**, specify the expiration date of the certificate. The expiration date can be found by running the following command: `openssl x509 -in <root crt file path> -text -noout` and then looking for the **Validity Not After** date.
-7. Click **Add**. The certificate is available.
-8. Go to **Universes -> Create Universe**. The **Create Universe** dialog appears.
-9. Configure the universe as desired.
-10. Based on your needs, select **Enable Node-to-Node TLS** and **Enable Client-to-Node TLS**.
-11. Choose an existing certificate from the **Root Certificate** drop-down list and then select the certificate that was uploaded.
-12. Create the universe.
+To view the certificate, use the list located at the top-right corner of the Yugabyte Platform console and click **Certificates**.
 
-## Connect to clusters
+## Self-Signed Self-Provided Certificates
 
-### Connecting to the YSQL endpoint with TLS
+Instead of using platform-provided certificates, you can also use your own self-signed certificates that you upload to the platform by following the procedure described in [How to Use Self-Signed Self-Provided Certificates to Enable TLS](#how-to-use-self-signed-self-provided-certificates-to-enable-tls). 
 
-If you enabled the Client-to-Node TLS option when you created your universe, then you must download client certificates to your client machine to connect to your database.
+The certificates must meet the following criteria: 
 
-1. Go to the **Certificates** page, then to your universe’s certificate, and select the **Download YSQL Cert** option as shown below.
+- Be in the `.crt` format and the private key must be in the `.pem` format, with both of these artifacts available for upload.
+- Contain IP addresses of the target database nodes or DNS names as the Subject Alternative Names (wildcards are acceptable). 
 
-![Download YSQL Certificate](/images/yp/encryption-in-transit/download-ysql-cert.png)
+The platform produces the node (leaf) certificates from the uploaded certificates and copies the certificate chain, leaf certificate, and private key to the nodes in the cluster. 
 
-This will download two files: `yugabytedb.crt` and `yugabytedb.key`.
+### How to Use Self-Signed Self-Provided Certificates to Enable TLS
 
-2. For testing with a `ysqlsh` client, copy these files to `<home-dir>/.yugabytedb` directory and change the permissions to `0600`.
+You can enable TLS using your own certificates, as follows:
+
+- Select **Certificates** from the list located at the top-right corner of the Yugabyte Platform console.
```

Comment:
@lizayugabyte can you change steps 1-2 to these? 
1. Navigate to Configs, select the Security tab, and then select the Encryption in Transit tab.

2. Click **Add Certificate** to open the **Add Certificate** dialog.
3. Select **Self Signed** certificate